modules/system/twig/Extension.php

```diff
@@ -55,6 +63,7 @@ public function getFilters()
         $filters = [
             new TwigSimpleFilter('app', [$this, 'appFilter'], ['is_safe' => ['html']]),
             new TwigSimpleFilter('media', [$this, 'mediaFilter'], ['is_safe' => ['html']]),
+            new TwigSimpleFilter('asset', [$this, 'assetFilter'], ['is_safe' => ['html']]),
```

Comment:
Can you remove the is_safe flags from the filters as well so that the function and filter behaviour matches?

Reply:
Done in https://github.com/wintercms/winter/pull/428/commits/83854069beca193a0beb1790a50bb9eef5aad001 :)